tests/DucksNet.IntegrationTests/PetsControllerTests.cs

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Do we have 0 tests for this controller? Then why do we have the file?

Reply:
^